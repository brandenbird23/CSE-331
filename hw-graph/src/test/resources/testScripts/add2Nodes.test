# add2Nodes Test File
# tests if 2 nodes can be added to a graph

# Create a graph
CreateGraph graph1

# Create 2 nodes
AddNode graph1 node1
AddNode graph1 node2

# print the nodes in graph
ListNodes graph1