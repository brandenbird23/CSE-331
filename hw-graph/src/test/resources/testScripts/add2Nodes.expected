# add2Nodes Test File
# tests if 2 nodes can be added to a graph

# Create a graph
created graph graph1

# Create 2 nodes
added node node1 to graph1
added node node2 to graph1

# print the nodes in graph
graph1 contains: node1 node2